### ISSUE TITLE
Opening a note pauses for multiple seconds

### ISSUE THREAD

**Issue by Cykelero** [sballin/alfred-search-notes-app#81]
On my Mac, when opening a note, Alfred usually freezes for 5-20 seconds after pressing enter; before it goes away and the Notes app is focused, selecting the note.

This isn't always the case: it seems that if the workflow was used very recently, the delay usually doesn't occur. (or, maybe, that's when the Notes app itself has been recently used, through the workflow or otherwise)

This happens regardless of whether Notes is hidden (⌘H) or not.

Keeping an eye on the Alfred workflow debugger, it seems that the “Show note/folder or create note” AppleScript action itself is stalling.

**To Reproduce**

Steps/context to reproduce the behavior:

1. Search for a note using its title
2. Press enter to reveal it in Notes
3. Alfred sometimes freezes before the workflow continues

**Debugger output**

```
[...]
[15:29:16.634] Search Notes[Script Filter] Processing complete
[15:29:16.639] Search Notes[Script Filter] Passing output 'F1DF3F39-711D-4907-A605-446B5C4841F0,x-coredata://526BFCA4-DE6E-41D4-AF6A-C7587175957B/ICNote/p5290,null,x-coredata://526BFCA4-DE6E-41D4-AF6A-C7587175957B/ICFolder/p5312,yi' to Run NSAppleScript
[15:29:23.531] Search Notes[Run NSAppleScript] Processing complete
[15:29:23.546] Search Notes[Run NSAppleScript] Passing output '' to Run Script
```

**Info**

Fill out this information if it's not already included in the error text.

 - macOS version: 14.3.1 (23D60)
 - CPU: Apple M1 Pro
 - Alfred version: 5.5
 - Workflow version: v3.6.2
 - In System Preferences > Security & Privacy > Privacy, there are checkmarks for Accessibility > Alfred 4, Full Disk Access > Alfred 4, Automation > Alfred 4 > Notes, and Automation > Alfred 4 > System Events? Yes to all four
 - What type of account are your notes stored in (iCloud/On My Mac/Email)? All of them in iCloud

**Comment by sballin** [comment]
Does the problem persist if you change the showEnclosingFolder environment variable to 0?

**Comment by Cykelero** [comment]
Thanks for the suggestions!
I tried both changes, but the issue seems to persist.

**Comment by Cykelero** [comment]
Right, it's likely a Notes bug. Hopefully it'll go away on its own at some point.

And thanks for making this workflow—I use it all the time. Couldn't live without it anymore!
